gcloud/bigquery/test_job.py

```diff
@@ -88,8 +87,6 @@ def _verifyInitialReadonlyProperties(self, job):
     def _verifyReadonlyResourceProperties(self, job, resource):
         from datetime import timedelta
 
-        self.assertEqual(job.job_id, self.JOB_ID)
```

Comment:
The `job_id` property is redundant:  it was left over from a time when I imagined that it would be generated:  instead, we require that the user pass in a name for the job (part of its URL).